actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/test/java/org/sunbird/common/models/util/fcm/FCMNotificationTest.java

```diff
@@ -41,15 +42,16 @@
   HttpClients.class,
   URL.class,
   BufferedReader.class,
-  HttpUtil.class,
+  HttpClientUtil.class,
   System.class,
   Notification.class
 })
 @PowerMockIgnore({"javax.management.*", "javax.net.ssl.*", "javax.security.*"})
 public class FCMNotificationTest {
 
-  @Test
+  //@Test
```

Comment:
Shouldn't these be in uncommented state? Pls add description so as to aid in review.

Reply:
uncommented the test case